guillotina/tests/test_annotations.py

```diff
@@ -94,3 +96,17 @@
         assert [k async for k in bucket.iter_keys(ob)] == [str(i) for i in _range]
         assert [v async for v in bucket.iter_values(ob)] == [i for i in _range]
         assert [(k, v) async for k, v in bucket.iter_items(ob)] == [(str(i), i) for i in _range]
+
+
+async def _test_bucket_dict_value_many_values(dummy_guillotina):
```

Comment:
test is commented?

Reply:
It's just to test speed. All functionality is already tested.